### ISSUE TITLE
Doc

### ISSUE THREAD

**Issue by MahiBegoug** [VirtualGoat/Blog-Website#1]
Please add readme file to explain your project ,I'm beginner in django

**Comment by VirtualGoat** [comment]
Yeah, sure. This project is ongoing and will be completed in 3-4 days. I'll add a readme file once it is completed.
